components/Events.js

```diff
@@ -6,6 +6,15 @@ const filters = {
 };
 
 export const Events = [
+	{
+		title: "Intern/UG Research Panel",
+		location: "WCH 205/206",
+		color: filters["Meetings"],
+		start: new Date(2022, 8, 27, 16, 0, 0),
+		end: new Date(2022, 8, 27, 18, 0, 0),
+		description:
+			"Are you someone searching for internships or research opportunities? For our first GM, we’ll be holding our annual intern & research panel. This is a great opportunity to ask questions and hear advice from students that have landed internship and undergraduate research positions.",
```

Comment:
Learn about the tips and tricks undergraduates use to secure internships and research opportunities.